src/Factory/CreditMemoFactory.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
What about license blocks? Here and in interface? We don't use them here?

Reply:
We don't have them in almost the entire project, but we probably should 🤔

Reply:
Should be handled by the coding standard tool.